relay-genesis/mock-difficulty.js

```diff
@@ -0,0 +1,23 @@
+const BitcoinRpc = require('bitcoind-rpc')
```

Comment:
I'm wondering if there is a better name for this file. As I understand it's setting bitcoin network difficulty in the MockRelay contract. Maybe `update-mock-relay.js`?